relayer/src/link/relay_path.rs

```diff
@@ -261,12 +261,16 @@ impl RelayPath {
         Err(LinkError::OldPacketClearingFailed)
     }
 
+    pub fn clear_packets(&self) -> bool {
```

Comment:
How about we keep the `clear_packets` name for the method that does the clearing, and renaming this to?

```suggestion
    pub fn will_clear_packets(&self) -> bool {
```